### ISSUE TITLE
Express no longer automagically detects vash

### ISSUE THREAD

**Issue by meandmycode** [kirbysayshi/vash#60]
In previous versions of vash registration with express was as simple as:

```js
var app = express();
app.set('view engine', 'vash');
```

In 0.8 this no longer seems to work but can be mitigated with the following code:

```js
vash.__express = vash.renderFile;
var app = express();
app.set('view engine', 'vash');
```

It seems like in 0.8, the __express export no longer exists.

Thanks,

**Comment by kirbysayshi** [comment]
Thanks for reporting! It will be fixed in the next patch release.

**Comment by kirbysayshi** [comment]
This has been released as v0.8.1!
